layouts/partials/elements/r-benefits.html

```diff
@@ -0,0 +1,33 @@
+{{ if .benefitsEnabled }}
+
+{{- $announcements := partial "helpers/get-locale-data" "announcements" -}}
+
+{{ with $announcements.benefits }}
+<r-benefits>
```

Comment:
It's a bit weird to have an element named like this, but it works. I get that you just copied the code for the newsletter, and perhaps it makes sense to do it in the same way. However, with the current structure it's not possible to add the benefits bar anywhere else than the main layout. I would bet that at some point someone wants to use it on a collection page or whatever.

For background, the newsletter is an element named `r-newsletter` because my intention was to make it a custom element. But that didn't happen, at least yet.

If you think this is good like this then no need to change, just wanted to give a bit of context/input.

Reply:
Yes, I just copied `r-newsletter` and tried to follow same pattern. Maybe it makes more sense to change this to section.
Fixed.